main/templates/main/homepage.html

```diff
@@ -1,7 +1,6 @@
-<!DOCTYPE html>
+{% extends "main/base_template.html" %}
+{% block content %}
 {% load static %}
-<html style="font-size: 16px;">
-
 <head>
```

Comment:
Do we need to keep the head and body tags?

Reply:
I've removed the head and body and put it all into "container" for keeping using the class and other stylings.